src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -383,13 +273,11 @@ public SignatureProviderFactory SignatureProviderFactory
         public override bool CanReadToken(string tokenString)
         {
             if (tokenString == null)
-            {
-               LogHelper.Throw(string.Format(CultureInfo.InvariantCulture, LogMessages.IDX10000, "tokenstring"), typeof(ArgumentNullException), EventLevel.Verbose);
-            }
+                throw LogHelper.LogArgumentNullException("tokenString");
 
             if (tokenString.Length * 2 > this.MaximumTokenSizeInBytes)
             {
-                IdentityModelEventSource.Logger.WriteInformation(string.Format(CultureInfo.InvariantCulture, LogMessages.IDX10719, tokenString.Length));
+                IdentityModelEventSource.Logger.WriteInformation(LogMessages.IDX10719, tokenString.Length);
                 return false;
             }
 
```

Comment:
It will be inconsistent to allow 100ms (JwtSecurityToken constructor) at one place and 2 seconds (CanReadToken method) at another. We should either change both or none.